cli/cmd/generate_gcp.go

```diff
@@ -54,6 +52,9 @@ var (
 	QuestionGcpAnotherAdvancedOpt      = "Configure another advanced integration option"
 	GcpAdvancedOptLocation             = "Customize output location"
 	QuestionGcpCustomizeOutputLocation = "Provide the location for the output to be written:"
+	QuestionGcpCustomFilter            = "Specify a custom filter which will supersede all other filter options"
+	QuestionGcpGoogleWorkspaceFilter   = "Filter out Google Workspace login logs from GCP Audit Log sinks?"
+	QuestionGcpK8sFilter               = "Filter out GKE logs from GCP Audit Log sinks?"
```

Comment:
We should probably invert this. Realistically we want this off by default. In fact I wonder if we should include these two as I'm not sure the ingest team supports ingesting these events yet

Reply:
`terraform-gcp-audit-log` has these as default True.  I'd the change to start in the TF module, keep flow of variable/parameter logic consistent.  And with a new Jira ticket to track this.

Reply:
I'm not sure that there needs to be a change in the Terraform module. I just meant that we probably should invert the question here or even go a step further and remove this from the CLI survey

Reply:
Removing this from survey.